sbc_compassion/tools/import_letter_functions.py

```diff
@@ -21,13 +21,117 @@
 import checkboxreader as cbr
 import bluecornerfinder as bcf
 import numpy as np
+from math import ceil
 from glob import glob
 from wand.image import Image
 
+from openerp import _
+
 
 ##########################################################################
 #                           GENERAL METHODS                              #
 ##########################################################################
+def testline(env, line_vals, csv_file, doc_name):
+    """ Test each line and verify the data given by the csv.
```

Comment:
Do you document anywhere what the csv file looks like ? Is it the same as the one used for the tests ? There should be at least somwhere inside the module a explanation about it (if there already is one, you can point to it inside this docstring).